test/e2e/framework.go

```diff
@@ -18,7 +18,6 @@ package e2e
 
 import "github.com/onsi/ginkgo/v2"
 
-// SIGDescribe annotates the test with the SIG label.
-func SIGDescribe(text string, body func()) bool {
-	return ginkgo.Describe("[kubernetes-sigs] "+text, body)
+func NFDDescribe(args ...interface{}) bool {
+	return ginkgo.Describe("[k8s-sigs/node-feature-discovery]", args...)
```

Comment:
If we want a top-level ginkgo label covering all nfd e2e tests we can specify it here to avoid repeating it all over

```suggestion
	return ginkgo.Describe("[k8s-sigs/node-feature-discovery]", append(args, ginkgo.Label("nfd-all"))...)
```